util/src/crypto_utils.cpp

```diff
@@ -1,6 +1,6 @@
 // Concord
 //
-// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Please move the content of this file to `crypto.cpp`

Reply:
Changes done https://github.com/arc-vmware/concord-bft/pull/12/commits/6aaca0e61da69ebbaa897378ee75083aff98d4e4